src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -163,12 +167,20 @@ const Query: FC<PipeProp> = ({Context}) => {
         </Suspense>
       </Context>
     ),
-    [RemoteDataState.Loading, query.text, updateText, editorContext.editor]
+    [
+      RemoteDataState.Loading,
+      query.text,
+      updateText,
+      editorContext.editor,
+      variables,
+    ]
   )
 }
 
 export default ({Context}) => (
-  <EditorProvider>
-    <Query Context={Context} />
-  </EditorProvider>
+  <VariablesProvider>
```

Comment:
move this up to the pipe component so all panels will have access to their own variable scope as that concept is fleshed out

Reply:
done.